### PR TITLE
Add quicktext button

### PR DESCRIPTION
**Pull request by mrjacobbloom**
See #205

Add a button to easily add text to empty c-blocks

### CODE REVIEW

File: resources/js/blocks.js [hunk 1]
```diff
@@ -327,6 +327,12 @@ SCRIPTING_AREA.addEventListener('contextmenu', function(ev) {
     }
 });
 
+SCRIPTING_AREA.addEventListener('click', function(ev) {
```
Comment (mrjacobbloom) [inline resources/js/blocks.js:330]: This is what's causing the merge conflict. #236 also added stuff to this line

Comment (mrjacobbloom) [inline resources/js/blocks.js:330]: (I needed to figure out how to pull from upstream, I think I got that figured out)

Comment (matthewr6) [inline resources/js/blocks.js:330]: `git pull origin branch-name`

Comment (mrjacobbloom) [inline resources/js/blocks.js:330]: but I have my own fork so I added this repo as a separate remote

Comment (matthewr6) [inline resources/js/blocks.js:330]: Just use branches from now on maybe

**Comment by mrjacobbloom** [comment]
Aaaaach this is a mess, I'm gonna redo these changes from a fresh branch

**Pull request closed without merging**